typings/blockly.d.ts

```diff
@@ -21560,15 +21554,16 @@ declare module "field_image" {
          * @type {boolean}
          * @protected
          */
-        protected isDirty_: boolean;
+        public isDirty_: boolean;
```

Comment:
Why is this public?

Reply:
it's public in the base class and typescript doesn't like when a child class changes the visibility https://github.com/google/blockly/issues/5803